docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -13,17 +13,22 @@ it and join the cluster.
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- AWS account with EC2 instances and permissions to create and attach IAM policies as well as permission to create Systems Manager documents.
-- EC2 instances running Ubuntu/Debian/RHEL/Amazon Linux 2 if making use of the default Teleport install script. (For other Linux distributions, you can install Teleport manually.) 
+- AWS account with EC2 instances and permissions to create and attach IAM
+policies.
+- EC2 instances running Ubuntu/Debian/RHEL/Amazon Linux 2 if making use of the
+default Teleport install script. (For other Linux distributions, you can
+install Teleport manually.)
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Create an EC2 invite token
+## Step 1/6. Create an EC2 invite token
 
 When discovering EC2 instances, Teleport makes use of IAM invite tokens for
-authenticating joining Nodes.
+authenticating joining Nodes. Read [Joining Nodes via AWS IAM Role](../../management/guides/joining-nodes-aws-iam.mdx)
```

Comment:
What additional information should the reader expect to get out of the linked guide? If it's further reading for general edification, I think the link should go in "Next steps" instead of this section.